tests/e2e_all/scripts/common/run_tests.sh

```diff
@@ -42,12 +42,15 @@ if [ $allowParallelization == "true" ] && command -v env_parallel >/dev/null 2>&
   export -f run_tests_for_daemon
   # Run a round of tests against each daemon in parallel
   parallel --jobs 5 \
+    --timeout 3m \
```

Comment:
This ends up being 3minutes per daemon - which may be a bit short, we will have to see.